src/gui/updater/ocupdater.cpp

```diff
@@ -283,9 +283,11 @@ void OCUpdater::slotStartInstaller()
             return QDir::toNativeSeparators(path);
         };
 
-        QString command = QString("&{msiexec /norestart /passive /i '%1'| Out-Null ; &'%2'}")
-            .arg(preparePathForPowershell(updateFile))
-            .arg(preparePathForPowershell(QCoreApplication::applicationFilePath()));
+        auto msiLogFile = cfg.configPath() + "msi.log";
```

Comment:
Is `configPath()` here a good choice for windows?

Reply:
I'd say so: it's next to the .msi file itself